js/src/index.ts

```diff
@@ -5,6 +5,7 @@ import { CloudflareToolSet } from "./frameworks/cloudflare";
 import { VercelAIToolSet } from "./frameworks/vercel";
 import { LangGraphToolSet } from "./frameworks/langgraph";
 import { Workspace } from "./env/index";
+import { ConnectionRequest } from "./sdk/models/connectedAccounts";
```

Comment:
Consider adding a brief comment above the import statement explaining the purpose of `ConnectionRequest` class and when it should be used. This would help SDK users understand its role in connection management.

Reply:
Dummy class that's returned. Used by user for testing. No other class are present.